.pre-commit-config.yaml

```diff
@@ -42,9 +42,19 @@ repos:
         name: isort (pyi)
         types: [pyi]
 -   repo: https://github.com/psf/black
-    rev: 24.8.0
+    rev: 24.10.0
     hooks:
     -   id: black
         args:
             - "--config"
             - "./pyproject.toml"
+-   repo: https://github.com/DavidAnson/markdownlint-cli2
+    rev: v0.16.0
+    hooks:
+    -   id: markdownlint-cli2
+        name: markdownlint
+        files: "docs/*.md"
```

Comment:
That's probably wrong, but it complained that the other one wasn't a valid regex, and now this one gives a warning that it's a regex and not a glob 🤷